### PR TITLE
Add an example Jobseeker to database seeds

### PR DESCRIPTION
**Pull request by david-mears-dfe**
Do we mind if the public can technically find out how to log in to our review apps?

If so, we will need to use environment variables for the username and/or password.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -216,3 +216,7 @@
             dsi_data: { "school_urns" => %w[137138 144523], "trust_uids" => %w[16644], "la_codes" => %w[852] },
             family_name: "Carter",
             given_name: "Valentine")
+
+Jobseeker.create(email: "jobseeker@example.com",
+                 password: "password",
```
Comment (cpjmcquillan) [inline db/seeds.rb:221]: Does this work? I'd be doubtful, surely the hashed password is what needs to be saved in the database?

Comment (cpjmcquillan) [reply db/seeds.rb:221]: It does work 🤔

Comment (csutter) [reply db/seeds.rb:221]: Devise magic! It hashes the password on save 🧙

**Review by csutter — APPROVED** [review]
Let's merge this for now (there's no damage anyone could cause) and revisit later.

**Pull request merged**